src/main/proto/th2_grpc_common/common.proto

```diff
@@ -193,12 +194,14 @@ message MessageFilter {
   string direction = 2 [deprecated = true];
   map<string, ValueFilter> fields = 3;
   ComparisonSettings comparison_settings = 4;
+  google.protobuf.StringValue description = 5;
```

Comment:
The description should be added to the RootMessageFilter because it matches the message we want to check